### DISCUSSION TITLE
make stack Leniency 0 disable it

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#20214] [Ideas]
currently stack leniency 0 still allows for the notes to stack if they're on the same timestamp. 
sometimes I just want no stacks
also yes this is an actual hindrance to mapping believe it or not.
![image](https://user-images.githubusercontent.com/78441708/189464983-10b87be6-b7c3-4a09-bd93-f3342f6ddc1f.png)

**Comment by peppy** [comment]
You are suggesting having two hitcircles at the same point in time with no way to visually discern. It's not valid?

**Reply by nagi-desuuu** [reply]
I'd agree.
@Axilotl17 what you're suggesting is making [Stack Zero](https://osu.ppy.sh/wiki/en/Beatmapping/Mapping_techniques/Unrankable) a thing in Lazer, which is usually an unrankable mapping technique only used for special occasions.
I would say it's not worth the combo breaks either way.

**Reply by Axilotl17** [reply]
sorry, what I would use this for is slider-stream conversion; When making streams, I'll often start a new stream on the end of the previous one, which overlaps at the same time that that one ends, or I'll use it when copying exact jump angles or jump distances. the slight stack actually messes up this perfect distance snap and causes slight errors. you can actually notice it when creating streams. I didn't consider just having 2 notes actually concurrent in a map when requesting this, I was just thinking in utility terms for mapping.

**Reply by Axilotl17** [reply]
It might not seem intuitive but I am willing to show you exactly how me and many others do it. an alternative to this is to handle the stacking differently, but I'm not sure exactly how to describe how It could be fixed.

**Reply by pha-Z** [reply]
im in the same camp as Axilotl17 here. in osu stable, i usually use the setting `View > Stacking (disabled)` because it is a nuisance during mapping. i commonly create helper objects for geometries and also like OP use helper slider objects to align stream segments. stacks are just irritating in those cases where i need precision.

IMO it would be good enough if was possible to disable Stacking under `View`  - just like in stable
